libcontainer/rootfs_linux.go

```diff
@@ -86,20 +85,19 @@ func prepareRootfs(pipe io.ReadWriter, iConfig *initConfig, mountFds mountFds) (
 		cgroupns:        config.Namespaces.Contains(configs.NEWCGROUP),
 	}
 	for i, m := range config.Mounts {
-		entry := mountEntry{Mount: m, idmapFD: -1}
+		entry := mountEntry{Mount: m}
 		// Just before the loop we checked that if not empty, len(mountFds) == len(config.Mounts).
```

Comment:
This is from https://github.com/opencontainers/runc/pull/3717, it is not len(mountFds), but len(mountFds.sourceFds). I think line 89-90 could be removed.